server/src/main/kotlin/com/xebia/functional/xef/server/models/Database.kt

```diff
@@ -0,0 +1,16 @@
+package com.xebia.functional.xef.server.models
+
+import kotlinx.datetime.Instant
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class XefTokens(
```

Comment:
Should we move this class to `XefTokensTable.kt`?

I know that it is a little bit different to others because of multiple primary keys, but I think that we should have this class with the table like `UsersTable` and `User`

Reply:
Oook, seems good. Updated!